### PR TITLE
Also show cases owned by users' locations or their children

### PR DESCRIPTION
**Pull request by esoergel**
@benrudolph
http://manage.dimagi.com/default.asp?185992

### CODE REVIEW

File: corehq/apps/reports/standard/cases/basic.py [hunk 1]
```diff
@@ -122,10 +123,10 @@ def case_owners(self):
         selected_reporting_group_ids = EMWF.selected_reporting_group_ids(self.request)
         selected_sharing_group_ids = EMWF.selected_sharing_group_ids(self.request)
 
-        # Show cases owned by any selected locations or their children
-        location_owner_ids = get_locations_and_children(
-            EMWF.selected_location_ids(self.request)
-        ).location_ids()
+        # Show cases owned by any selected locations, user locations, or their children
+        loc_ids = (EMWF.selected_location_ids(self.request) +
+                   get_users_location_ids(self.domain, selected_user_ids))
```
Comment (czue) [inline corehq/apps/reports/standard/cases/basic.py]: tiny but should we deduplicate these two lists?

**Comment by czue** [comment]
my comment isn't blocking - this is sweet!

File: corehq/apps/reports/standard/cases/basic.py (hunk 1 above)
Comment (esoergel) [inline corehq/apps/reports/standard/cases/basic.py]: https://github.com/dimagi/commcare-hq/pull/8913

**Comment by benrudolph** [comment]
:+1: sorry missed this yesterday

**Pull request merged**